invenio_records_resources/services/records/service.py

```diff
@@ -398,6 +398,7 @@ def _read_many(
         search_opts=None,
         extra_filter=None,
         preference=None,
+        sort=None,
```

Comment:
isn't sort passed in search_opts ? if you add it to the search configuration on the correct service (check RDMSearchOptions for reference) it should work without adding this kwarg

Reply:
Agreed to keep the existing solution as the alternative would require some refactoring between the different methods of searching like search, read_many, read_all.... Vocabularies are using directly the `_read_many()` method.